### ISSUE TITLE
FTBFS: tigerbeetle

### ISSUE THREAD

**Issue by imjasonh** [wolfi-dev/os#23352]
```
/home/build/build.zig:1014:22: error: expected 2 arguments, found 3
/home/build/build.zig:1039:22: error: expected 2 arguments, found 3
/home/build/build.zig:1091:22: error: expected 2 arguments, found 3
```

**Comment by smoser** [comment]
fixed [#23605](https://github.com/wolfi-dev/os/pull/23605)

**Comment by ritster** [comment]
Issue wasn't fixed in previous PR (not actually merged) - now fixed in https://github.com/wolfi-dev/os/pull/25089. The upstream build process changed, now our package config reflects that.
